Scripts/BuildDatabank/AddData.py

```diff
@@ -309,32 +309,44 @@
         " the headgroup is used."
     )
 
-    top = ""
-    traj = ""
-
-    if sim["SOFTWARE"] == "gromacs":
-        top = os.path.join(dir_tmp, sim["TPR"][0][0])
-        traj = os.path.join(dir_tmp, sim["TRJ"][0][0])
-    elif sim["SOFTWARE"] == "openMM" or sim["SOFTWARE"] == "NAMD":
-        traj = os.path.join(dir_tmp, sim["TRJ"][0][0])
-        top = os.path.join(dir_tmp, sim["PDB"][0][0])
-    else:
-        logger.error(
-            "SOFTWARE '%s' is not a proper option.\n"
-            "Use either 'gromacs', 'openMM', or 'NAMD'.")
+    try:
+        struc, top, traj = get_3major_fnames(sim, joinPath=dir_tmp)
+    except (ValueError, KeyError) as e:
+        logger.error(str(type(e)) + " => " + str(e))
         quit(1)
+    except Exception as e:
+        logger.error("Unkonwn error during `get_3major_fnames..`")
+        logger.error(str(type(e)) + " => " + str(e))
+        quit(4)
 
     leaflet1 = 0  # total number of lipids in upper leaflet
     leaflet2 = 0  # total number of lipids in lower leaflet
 
-    gro = os.path.join(dir_tmp, "frame0.gro")
-
+    # try to generate zero-frame structure from top + trajectory
+    failFromTop = False
```

Comment:
As I mentioned before, PEP recommends to name variables in the snakecase: `fail_from_top`. There are several places in this PR where the camelcase is used. Not sure that it is an urgent thing, but it would be great to follow the decided naming guidelines at least in all new code.

Reply:
done